src/mmw/js/src/analyze/templates/catchmentWaterQualityTable.html

```diff
@@ -31,8 +29,8 @@
       <tfoot>
           <tr class="catchment-water-quality-table-footer">
               <td></td>
```

Comment:
Looks like you missed deleteing the empty `<td></td>` here (and in point src), so the `colspan` is offset:

![screenshot from 2016-11-01 18 11 21](https://cloud.githubusercontent.com/assets/1014341/19909541/9e294a02-a05e-11e6-8671-b16a7c6cc515.png)